models/ai85net-actiontcn.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         self.num_classes = num_classes
         self.cnn_out_shape = (1, 1)
         self.cnn_out_channel = 32
+        self.num_frames = 15
```

Comment:
a little more docstring would be great for the init, especially noting for setting num frames statically to 15 and maybe why

Reply:
Thanks, I will add an explanation for this.